3-ecommerce-db/projects/server.js

```diff
@@ -0,0 +1,71 @@
+const express = require("express");

```

Comment:
i think this file should be updated with the correct logic in it, applicable to `cyf-ecommerce` task ;)